LayoutTests/imported/w3c/web-platform-tests/trusted-types/worker-constructor.https.html

```diff
@@ -64,7 +64,16 @@
 // Tests with default policy.
 promise_test(t => {
   trustedTypes.createPolicy("default", {
-      createScriptURL: s => s.replace("potato", "https") });
+    createScriptURL: (s, _, sink) => {
+      if (s === "Worker") {
+        assert_equals(sink, 'Worker create');
+      } else if (s === "SharedWorker") {
+        assert_equals(sink, 'SharedWorker create');
+      } else if (s == "service_worker") {
+        assert_equals(sink, 'ServiceWorkerContainer addRegistration');
```

Comment:
`ServiceWorkerContainer register` I think it should be this? Per https://w3c.github.io/ServiceWorker/#navigator-service-worker-register

If webkit uses addRegistration currently that's due to a bug, and will be fixed by https://github.com/WebKit/WebKit/pull/27880